### PR TITLE
Add playlist filtering

### PR DESCRIPTION
**Pull request by maxenced**
This PR add support for playlist filtering.

Right now, there is only an inclusive filter (ie, only playlists *matching* pattern will be included).
It supports multiple filters (expect a logical OR here, playlist will be included if they match *at least one* filter).

Tests are failing now, but can't make them work on master neither.

This solves https://github.com/custom-cards/spotify-card/issues/140

### CODE REVIEW

**Comment by maxenced** [comment]
Anything I can do to make this move forward ? cc @fondberg @FL550

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
   "dependencies": {
     "custom-card-helpers": "^1.6.4",
     "home-assistant-js-websocket": "^5.6.0",
-    "lit-element": "^2.3.1"
+    "lit-element": "^2.3.1",
+    "jsonpath-plus": "^5.0.7"
   },
```
Comment (fondberg) [inline package.json:24]: Why is this needed?

Comment (maxenced) [reply package.json]: You're right, this is not needed, I first tried to allow jsonpath to specify keys so that you was able to reach nested attributes. But it makes things overcomplicated so finally not used it.

**Review by fondberg — APPROVED** [review]
Sorry for the delay! I'm not in this repo that often. Will merge and make a release
